install/update.php

```diff
@@ -8683,6 +8683,9 @@ function migrateTablesFromFusinvDeploy($migration)
          //"deployorders fixer : final order structure for ID ".$order_config['id']."\n" .
        //   json_encode($json_order,JSON_PRETTY_PRINT) ."\n"
        //);
+        $migration->addField('glpi_plugin_glpiinventory_taskjobs', 'restrict_to_task_entity', 'bool', ['value'   => '1']);
+        $migration->migrationOneTable('glpi_plugin_glpiinventory_taskjobs');
+
         $pfDeployPackageItem = new PluginGlpiinventoryDeployPackageItem();
         $pfDeployPackageItem->updateOrderJson($order_config['id'], $json_order);
```

Comment:
IMHO, The best way to fix this is to do the `json_encode()` + `$DB->update()` operations directly here, to prevent GLPI core hooks to be executed.

Reply:
agree with that (see my last commit)